src/main/java/edu/harvard/dbmi/avillach/dictionary/dashboarddrawer/DashboardDrawerRowMapper.java

```diff
@@ -0,0 +1,35 @@
+package edu.harvard.dbmi.avillach.dictionary.dashboarddrawer;
+
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Array; // For handling SQL Array
+import java.util.Arrays;
+import java.util.List;
+
+public class DashboardDrawerRowMapper implements RowMapper<DashboardDrawer> {
+
```

Comment:
I would at least make an abstract class for this that I could extend to fulfill multiple Dashboard Layouts.

Should probably do this for other Classes such as Dashboard.